SimpleRabbit.NetCore/Model/IDispatchModel.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace SimpleRabbit.NetCore
+{
+    /// <summary>
+    /// Model to implement in order to use an Ordered dispatcher
+    /// 
+    /// </summary>
+    /// <remarks>
+    /// The basic message will be used to perform channel operations.
+    /// 
+    /// </remarks>
+    public interface IDispatchModel
+    {
+        BasicMessage BasicMessage { get; set; }
```

Comment:
Note: The BasicMessage can be hidden from the user, and this interface would no longer be needed. Instead the ordered dispatcher could use MessageWrapper<T>